00-RELEASENOTES

```diff
@@ -1,16 +1,284 @@
+
+Behavior Changes - Cluster
+==========================
+* Allow `CLUSTER NODES/INFO/MYID/MYSHARDID` during loading state. (#596)
+* Make cluster replicas return `ASK` and `TRYAGAIN` during slot migration. (#495)
+
+Behavior Changes - Sentinel
+===========================
+* Replace `master-reboot-down-after-period` with `primary-reboot-down-after-period`
+  in `sentinel.conf`. (#647)
+
+Bug Fixes - Core
+================
+* Fix a bug that caused LRU/LFU inconsistencies for some integer objects. (#250)
+* Fix a bug where Valkey may use a sub-optimal encoding for some data types.
+  (Redis#13148)
+* Fix propagation of `entries_read` by calling `streamPropagateGroupID`
+  unconditionally. (Redis#12898)
+* Fix race condition issues between the main thread and module threads.
+  (Redis#12817)
+* Wake blocked clients ASAP in next `beforeSleep` for `WAITAOF`. (Redis#12627)
+* Fix crash in crash-report and improve thread management with RW locks.
+  (Redis#12623)
+
+Bug Fixes - Cluster
+===================
+* Fix a bug where a shard returns the incorrect slot slot information in
+  `CLUSTER SHARDS` command on primary failure. (#790)
+* Allow module authentication to succeed when the cluster is down. (#693)
+* Fix `PONG` message processing for primary-ship tracking during fail-overs.
+  (Redis#13055)
+* Prevent double freeing of cluster link with `DEBUG CLUSTERLINK KILL`.
+  (Redis#12930)
+* Unsubscribe all clients from replica for shard channel if the primary
+  ownership changes. (Redis#12577)
+
+Bug Fixes - CLI
+===============
+* Fix `valkey-check-aof` misidentifying data in manifest format as MP-AOF.
+  (Redis#12958)
+* Fix `valkey-cli` to respect the `--count` option without requiring
+  `--pattern`. (Redis#13092)
+
+Internal Codebase Improvements
+==============================
+* Enable debug asserts for cluster and sentinel tests. (#588)
+* Introduce a minimal debugger for Tcl integration test suite. (#683)
+* Set up clang-format GitHub action for automated code formatting checks. (#538)
+* Replace custom atomic logic with C11 _Atomics. (#490)
+* Add fast fail option for Tcl test cases. (#482)
+* Introduce a simple unit test framework. (#460)
+* An initial simple unit test framework. (#344)
+* Introduce Codecov for automated code coverage tracking. (#316)
+* Remove deprecated `redis-trib` CLI program. (#281)
+* Add `-fno-omit-frame-pointer` to default compilation flags to improve
+  debuggability. (Redis#12973)
+* Refactor the per-slot dict-array db.c into a new kvstore data structure.
+  (Redis#12822)
+* Unified database rehash method for both standalone and cluster modes.
+  (Redis#12848)
+* Clarify and decouple the sampling logic in eviction to improve readability.
+  (Redis#12781)
+* Rewrite large printf calls to smaller ones for readability. (Redis#12257)
+
+Experimental
+============
+* Introduce Valkey Over RDMA transport (experimental). (#477)
+
+We appreciate the efforts of all who contributed to this release:
```

Comment:
Can we sort them by contributions (commits or lines)? It seems that there are too many people at first glance, and some of them seem very minor, such as typo, we have also put them up.